### ISSUE TITLE
Error occurs when invalid timeout parameter passed into Rocky.on(verb, signature, callback[, timeout])

### ISSUE THREAD

**Issue by dbukhmastov** [electricimp/Rocky#23]
#### Scenario:
```squirrel
local app = Rocky();
app.onException(function(context, ex) {
    server.log("Got exception: " + ex);
    context.send(500);
});
app.on("GET", "/test", function(context) {
    context.send(200);
}.bindenv(this), /*timeoutRoute*/"invalid");
imp.wakeup(0, function() {
    local req = http.request("GET", http.agenturl() + "/test", {}, "body");
    req.sendasync(function(res){});
}.bindenv(this));
```
#### Log:
[Agent] ERROR: bad parameters to imp.wakeup(seconds, callback[, name])
[Agent] ERROR:   from _onrequest agent_code:178
#### Tests:
This issue affects class: tests/CoreRockyMethodNegative.nut method: testInvalidParamsTimeout()

**Comment by smittytone** [comment]
This will be fixed in Rocky 3.0.0

**Comment by smittytone** [comment]
Fixed in 3.0.0
